brave/brave-core
# Adds error logic for one-time tip

Resolves https://github.com/brave/brave-browser/issues/3052

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- claim grant
- go to verified site
- open panel
- select amount in donation banner
- (when you will click send tip you need to turn internet off immediately)
- click send tip
- disable internet
- make sure that you see error in the panel 
![untitled](https://user-images.githubusercontent.com/9574457/51903053-ad9b3f80-23bb-11e9-80f5-4f97efd107a0.jpg)
 

## Reviewer Checklist:
